docs/docs_requirements.txt

```diff
@@ -23,12 +23,17 @@ PyYAML==3.11
 South==1.0.2
 python-ldap==2.4.19
 
-# Deis client requirements
+# Deis client requirements, copied from client/requirements.txt
 docopt==0.6.2
+ndg-httpsclient==0.3.3
+pyasn1==0.1.7
+pyOpenSSL==0.15.1
 python-dateutil==2.4.2
+#PyYAML==3.11
```

Comment:
how come we're commenting out PyYAML here? Just because we don't want to handle the hassle of installing PyYAML's dependencies?